src/customJs/localization.test.js

```diff
@@ -0,0 +1,43 @@
+import { intl, setLocale } from './localization';
+
+describe(setLocale.name, () => {
+  it.each([{ lang: 'es-ES' }, { lang: 'en-US' }, { lang: 'fr-FR' }])(
+    'should call Unlayer.setLocale with right language',
+    ({ lang }) => {
+      // Arrange
+      const unlayerDouble = prepareUnlayerGlobalObject();
+
+      // Act
+      setLocale(lang);
+
+      // Assert
+      expect(unlayerDouble.setLocale).toHaveBeenCalledWith(lang);
+    },
+  );
+
+  it.each([
+    { lang: 'es-ES', expected_size_translation: 'Tamaño' },
+    { lang: 'en-US', expected_size_translation: 'Size' },
+    { lang: 'fr-FR', expected_size_translation: 'Tamaño' },
+  ])(
+    'should prepare intl with right language',
```

Comment:
```suggestion
    'should prepare intl using $lang language',
```

Reply:
Pienso que es mejor especificar por cada _**it**_ que lenguaje se debe usar, de lo contrario tendríamos 3 test con la misma descripción, eso es confuso. Usando el signo pesos podemos interpolar las variables del **_each_** de esta forma mejoramos la descripción.

Reply:
Buena idea @Cromeror! 

Yo [lo apliqué en el test de mi commit](https://github.com/FromDoppler/unlayer-editor/pull/312/commits/e4095679bc3c7a5e4afdac0b894da1e66ec8585c#diff-c4e8ac7f4f786fea2c5bc9bbc64d5db526e6d4d6e6f732bcd3e66b2953251973R21) y el resultado es mucho más claro (forcé un error):

<img width="527" alt="image" src="https://user-images.githubusercontent.com/1157864/163995641-b5c3ee33-46aa-4630-9733-06b9122802b9.png">

Reply:
Buenísimo, aplicado.